test/powershell/Modules/Microsoft.PowerShell.Utility/Set-PSBreakpoint.Tests.ps1

```diff
@@ -67,8 +67,8 @@ set-psbreakpoint -command foo
 
     It "-script and -line can take multiple items" {
         $brk = sbp -line 11,12,13 -column 1 -script $scriptFileName,$scriptFileName
-        $brk.Line | Should Be 11,12,13
-        $brk.Column | Should Be 1
+        $brk.Line | Should BeIn 11,12,13
```

Comment:
Above I see another pattern like `Should BeIn @(11,12,13)`. 
What pattern we'll use in tests?

Reply:
I don't think it matters - both are valid expressions

Reply:
We should probably just standardize on one of them for consistency/readability.  I prefer sans @(), myself.

Reply:
I agree that we will request `@()` template for `Should BeIn` in future test reviews.